src/Symfony/Component/Intl/Timezones.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl;
+
+use Symfony\Component\Intl\Exception\MissingResourceException;
+
+/**
+ * Gives access to timezone-related ICU data.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class Timezones extends ResourceBundle
+{
+    /**
+     * @return string[]
+     */
+    public static function getTimezoneCodes(): array
```

Comment:
`getZoneCodes` maybe ... @webmozart WDYT?

Reply:
Is there any official terminology for these identifiers?

Reply:
per http://icu-project.org/apiref/icu4j/com/ibm/icu/util/TimeZone.html i lean to `getTimezoneIds()`

PHP also confirms:

https://www.php.net/manual/en/intltimezone.getid.php
https://www.php.net/manual/en/function.date-default-timezone-set.php (`$timezone_identifier`)

Reply:
`getIds()` then?

Reply:
yes, it's not ambigious indeed ;) now fixed.